src/App.js

```diff
@@ -1,8 +1,20 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './App.scss';
+import Sum from './components/Sum';
 
-const App = () => (
-  <h1>React sum</h1>
+const App = ({ a, b }) => (
+  <section>
+    <h1>React sum</h1>
+    <Sum a={2} b={3} />
```

Comment:
{2} is of type Number, and in your proptypes you indicate that this component accepts 2 strings